src/Apps/Artwork/Components/ArtworkDetails/RequestConditionReport.tsx

```diff
@@ -0,0 +1,205 @@
+import { Button, Col, Modal, Row, Sans, Serif, Spinner } from "@artsy/palette"
+import React, { useState } from "react"
+import { commitMutation, graphql, RelayProp } from "react-relay"
+
+import { AnalyticsSchema, track, useSystemContext, useTracking } from "Artsy"
+import * as Schema from "Artsy/Analytics/Schema"
+import { ErrorModal } from "Components/Modal/ErrorModal"
+import * as authentication from "Components/NavBar/Utils/authentication"
+import createLogger from "Utils/logger"
+
+import { RequestConditionReportMutation } from "__generated__/RequestConditionReportMutation.graphql"
+
+const logger = createLogger(
+  "Apps/Artwork/Components/ArtworkDetails/RequestConditionReport"
+)
+
+export interface RequestConditionReportProps {
+  saleArtworkID: string
+  relay?: RelayProp
+}
+
+export const RequestConditionReport: React.FC<RequestConditionReportProps> = track(
+  {
+    context_module: AnalyticsSchema.ContextModule.RequestConditionReport,
+  }
+)(props => {
+  const { mediator, user } = useSystemContext()
+  const { trackEvent } = useTracking()
```

Comment:
Whenever I come back to this the amount of boilerplate around tracking is surprising. Are both a HOC with `track()()` and the `useTracking` hook necessary?

Reply:
Good point. Might be able to lose that HOC with `track()` here.